### ISSUE TITLE
File context menu doesn't have an exit item

### ISSUE THREAD

**Issue by triptych** [4ian/GDevelop#1498]
## Describe the bug
The GDevelop app does not have an exit option under File, like most apps. This is not just a cosmetic issue but an Accessibility bug.

## To Reproduce
1. Launch GDevelop
2. Choose "file"
3. observe menu.
4. unable to exit or close app.
( Note: Close Project just closes out the project windows, it doesn't close the GDevelop App - which is fine, but there ALSO needs to be an EXIT option)

* Please include a link to a game if possible!
* If applicable, add screenshots to help explain your problem.
Screenshot: 
![UOuGmUOcMu](https://user-images.githubusercontent.com/8691/76055239-eba45480-5f27-11ea-8735-95bb74f33179.png)

## Other details
Windows 10, Gdevelop app. Beta 89.

**Comment by ozata** [comment]
I produced the error on Ubuntu 18.04.4 LTS (Bionic Beaver). Since we are using electron, I believe it won't matter that I am doing this on GNU/Linux. Do you guys have any suggestions, nevertheless I will comb through the internet to find a solution.

**Comment by 4ian** [comment]
Something to add here: https://github.com/4ian/GDevelop/blob/master/newIDE/app/src/MainFrame/ElectronMainMenu.js

on macOS, the close should *not* be in the File menu

**Comment by mhk19** [comment]
@4ian i want to work on this issue. Can you please guide me?

**Comment by 4ian** [comment]
@mhk19 I don't have time to guide everyone - I need autonomy from you to search and can only have time to answer to precise technical questions.

Start here: https://github.com/4ian/GDevelop/blob/master/newIDE/app/src/MainFrame/ElectronMainMenu.js and investigate the existing buttons.

**Comment by NilayMajorwar** [comment]
Really sorry @mhk19, I was trying this one without mentioning on the thread, and saw your message a bit too late - my bad :sweat_smile:

**Comment by triptych** [comment]
@mhk19 take a look at the commit that fixed this to help you learn about the code so that you can be that much more up to speed when the next bug comes around.
